cognite/client/utils/_logging.py

```diff
@@ -62,11 +64,22 @@ def format(self, record: logging.LogRecord) -> str:
 
 
 def _configure_logger_for_debug_mode() -> None:
-    logger = logging.getLogger("cognite-sdk")
+    logger = logging.getLogger(COGNITE_LOGGER_NAME)
```

Comment:
Then this becomes just logging.getLogger("cognite.client") as all modules' `__name__` variable will start with "cognite.client".